spec/filters/platform/safari/literal_regexp.rb

```diff
@@ -1,6 +1,6 @@
 # NOTE: run bin/format-filters after changing this file
 opal_unsupported_filter "Literal Regexp" do
-  fails "Literal Regexps handles a lookbehind with ss characters"
-  fails "Literal Regexps supports (?<! ) (negative lookbehind)"
-  fails "Literal Regexps supports (?<= ) (positive lookbehind)"
+  fails "Literal Regexps handles a lookbehind with ss characters"
+  fails "Literal Regexps supports (?<! ) (negative lookbehind)"
+  fails "Literal Regexps supports (?<= ) (positive lookbehind)"

```

Comment:
This should not be the case in modern Safari

Reply:
Thanks, done.